docs/tools/appium/reference.md

```diff
@@ -70,7 +70,9 @@ public void setup() throws MalformedURLException {
 
 By default, the TestObject platform uninstalls the application, performs device cleaning, and de-allocates the device between every Appium session. This means that if you have multiple test methods, you will by default wait for this cleaning process between every test method.
 
-To speed up test execution, you can set the desired capability `testobject_cache_device` to `true`. This will leave the device allocated to you for a short time after each Appium session. If you immediately start another test on the device, you won't need to wait for the allocation and device cleaning process to be repeated.  
+To speed up test execution, you can set the desired capability `testobject_cache_device` to `true`. This will leave the device allocated to you for a short time after each Appium session. If you immediately start another test on the device, you won't need to wait for the allocation and device cleaning process to be repeated.
+
+The `testobject_cache_device` capability can be used in conjunction with the standard `noReset` Appium capability, which is set to false by default (your application will be uninstalled and reinstalled after every test, as per normal Appium behaviour). If `noReset` is set to true, the application you are testing won't be uninstalled after every test run. This might save you further time, but it won't be suitable for test setups which require the application's state to be reset between tests. Note that while `testobject_cache_device` is set, no device cleaning will take place before the end of the session, regardless of `noReset` value.
```

Comment:
- "(your application will be ... normal Appium behaviour)" can be taken out of parentheses and made into it's own sentence. I would then say, "If noReset is set to false, your application will be uninstalled ....". I think that this is more explicit, and the information is important enough to have it's own sentence.

- "Note that while `testobject_cache_device` is set, no device cleaning will take place before the end of the session, regardless of `noReset` value.". I think that we already made this clear with a whole paragraph  before this, and it is a confusing sentence. At least "is set" shout be changed to "is set to true". Also "before the end of the session" should be changed to "between your sessions".